Environment/NoiseLevelObserved/doc/spec.md

```diff
@@ -40,15 +40,15 @@ This entity is primarily associated with the Smart City and environment vertical
 It can be represented by an specific time instant or by an ISO8601 interval. As a workaround for
 the lack of support of Orion Context Broker for datetime intervals, it can be used two separate attributes: `dateObservedFrom`, `dateObservedTo`. 
     + Attribute type: [DateTime](https://schema.org/DateTime) or an ISO8601 interval represented as [Text](https://schema.org/Text). 
-    + Mandatory
+    + Optional
         
 + `dateObservedFrom` : Observation period start date and time. See `dateObserved`. 
     + Attribute type: [DateTime](https://schema.org/DateTime). 
-    + Optional
+    + Mandatory
```

Comment:
Please, be careful with stating attributes as Mandatory. It makes the model less usable

Reply:
I disagree. We need to have something mandatory, at least a minimum meaningful set of attributes. Otherwise cities would claim they are compliant with but without publishing anything relevant.

Reply:
Understood. I think it's a trade off between backward compatibility and meaningful attributes.